### ISSUE TITLE
Existence minimum (mínimo de existência) isn't being taken into account

### ISSUE THREAD

**Issue by josecoelhomelo** [franciscobmacedo/remotefreelancept#41]
Isn't the IRS exempt from payment under a certain yearly amount (mínimo de existência)?

E.g. for 2023 it was 10 640 € and for 2024 it's 11 480 €.

I can work on the project and make a pull request, though I want to be sure I'm making the right analysis.

**Comment by franciscobmacedo** [comment]
Yes you are right. When I created this tool, it was for a friend with a higher salary than the minimum amount, so I never bothered to handle that case. Fell free to make a PR! just add some sources of those values (mínimo de existência), so  we can validate that they are correct, if you don't mind.

**Comment by josecoelhomelo** [comment]
Sure, will do as soon as I can.
